packages/react-cards/src/components/Card/useCardClasses.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Either in this PR or later, it would be good to update this package to use the newer (`react` not `react-legacy`) eslint config and remove these disables.